### DISCUSSION TITLE
Bro, Take a look at the issues at my repository

### DISCUSSION THREAD

**Discussion by pranavajy** [warmachine028/Better-Calculator#7] [General]
I am a really a noob at python and I just started learning it recently,I started a repository with my basic knowledge . I need your help to take a look at the issues ,.

**Comment by warmachine028** [comment]
Yes I've seen the issues bro , but the problem is that i am not accustomed to SQLite very well. Actually I've been working with MySQL DBMS lately. I also did a [Hospital Management project](https://github.com/warmachine028/python_code_vault/blob/master/Hospital%20Management.py) in my high school but without database support . You can have some ideas there :)
